### PR TITLE
Typos, spelling and minor improvements.

### PR DESCRIPTION
**Pull request by kgish**
Took the readme on a quick spin and took the liberty to make some modifications, curious what you think.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,63 +50,66 @@ Start the server using:
...
 * `frontend/app/openproject-proto_plugin-app.js`
 * `frontend/app/controllers/kittens.js`
 * `app/views/kittens/index.html.erb`
 
-If you want to work within the frontend's angular app you will need to provide a `package.json`.
-Take a look at the `frontend` folder to see an example Angular controller which is used in the "kittens" index view.
+If you want to work within the frontend's AngularJS app you will need to provide a `package.json`. Take a look at the `frontend` folder to see an example Angular controller which is used in the "kittens" index view.
 
-Changes in the frontend require running webpack to update. For that run the following in the OpenProject folder (not the plugin):
+Any changes made to the frontend require running webpack to update. For that run, go to the OpenProject folder (NOT the plugin directory) and execute the following command:
```
Comment (machisuji) [inline README.md]: The "run, " doesn't make sense here anymore like this, does it?
I suggest changing the beginning of the sentence to "To do that go to the OpenProject folder ...".

**Review by machisuji — CHANGES_REQUESTED** [review]
That looks good. Thanks for taking the time to polish the README a little. I just have one minor remark.

File: README.md (hunk 1 above)
Comment (kgish) [reply README.md]: Makes sense, agreed!

Comment (machisuji) [reply README.md]: Ok, once you amended that this is good to merge! 👍

**Comment by machisuji** [comment]
Thanks!

**Pull request merged**